docs/reference/functions.md

```diff
@@ -264,3 +264,8 @@ example-value-or-externalValue:
       - externalValue
       - value
 ```
+<!-- 
```

Comment:
@philsturgeon Switching to custom function means I can drop that, as we only document core functions, right?

Reply:
Correct, we document the OpenAPI rules and the function is just an implementation detail.